packages/neuron-ui/src/components/Settings/TabBar.tsx

```diff
@@ -27,15 +27,15 @@ const TabBar = (props: any) => {
   const contents = ['General', 'Wallets', 'Network']
   const [selectedIndex, setSelectedIndex] = useState(0)
 
-  useEffect(() => {
-    props.history.push(tabs[0])
-  }, [])
-
   const handleAction = (index: number) => {
     setSelectedIndex(index)
     props.history.push(tabs[index])
   }
 
+  useEffect(() => {
```

Comment:
Will this navigate to tab 0 every time the page loads, even we intend to other tabs?

Reply:
When coming in settings page firstly, neuron would show general settings page (tab 0).

 If you want to navigate to other tabs, you need to click tab titles.

Reply:
1. if handleAction is to `props.history.push(tabs[index])`, why not set the link to setting as `tabs[index]`

2. we may need to other tabs directly, for example, when we clicked on the `network info` in the header, it may show the setting of the chain.

Reply:
If settings page (include general, wallets and network) is active, clicking tab titles will not call useEffect method. So when you want to navigate to other tabs, tab 0 will not always display.

Reply:
Yes, when we are at setting page, the tab bar will not be re-rendered on switching. But if we are at other pages, like wallet detail, and on clicking on the network in the header, it navigates us to wallet-setting, at that time, the tab bar is rendered and useEffect works to navigate us to general-setting.

Reply:
Ok. I see.